### PR TITLE
Remove removed scopes from default and fix unit tests

### PR DESCRIPTION
**Pull request by mikealmond**
Merges #53 and #54 with some additional fixes.

### CODE REVIEW

**Comment by H4wKs** [comment]
Thank you @mikealmond & @grifx :)

**Comment by SteppingHat** [comment]
Hey guys! Sorry to heckle you all but any ETA on the merge of this PR? I feel like a number of us have a hacky vendor file edit on prod as a workaround while we wait for #55 to be resolved 😅

**Comment by SteppingHat** [comment]
Hey guys! Sorry to heckle you all but any ETA on the merge of this PR? I feel like a number of us have a hacky vendor file edit on prod as a workaround while we wait for #55 to be resolved 😅 Thanks! ❤️

**Comment by H4wKs** [comment]
> Hey guys! Sorry to heckle you all but any ETA on the merge of this PR? I feel like a number of us have a hacky vendor file edit on prod as a workaround while we wait for #55 to be resolved 😅 Thanks! ❤️

Well, we can't do much more here than wait for the maintainer (@SammyK) to approve the PR. I have tried to reach out to him on LinkedIn as well, hopefully he will get the message but as can see, this package is not that active, the last release was 3 years ago.
Not sure if any other of @thephpleague members can step in and help here (ping @alcohol @Anahkiasen @chalasr )

**Comment by alcohol** [comment]
I can merge it... but to be honest I've never done this outside of the context of my own library. Not sure if I would be overstepping any boundaries here.

**Comment by alcohol** [comment]
Also willing to take ownership if that is truely nonexistent. But no idea what the process for this is.

**Comment by H4wKs** [comment]
> I can merge it... but to be honest I've never done this outside of the context of my own library. Not sure if I would be overstepping any boundaries here.

@alcohol thanks for your reply. I perfectly understand, I was hopping that you or someone from the team may be able to reach out more easily to @SammyK or someone else who would be confortable to do it.

**Comment by H4wKs** [comment]
@shadowhand any chance you could help here as you already working on oauth2-facebook & oauth2-client & oauth2-google  ? Or Sephster as you work on oauth2-server
I am afraid that if this is not solved shortly, people are going to stop using @thephpleague oauth2 package.

File: src/Provider/Facebook.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
         $fields = [
             'id', 'name', 'first_name', 'last_name',
             'email', 'hometown', 'picture.type(large){url,is_silhouette}',
-            'cover{source}', 'gender', 'locale', 'link', 'timezone', 'age_range'
```
Comment (shadowhand) [inline src/Provider/Facebook.php:106]: Does `cover` no longer exist in the API?

Comment (shadowhand) [reply src/Provider/Facebook.php:106]: Ah, I see #54.

**Comment by shadowhand** [comment]
Released in version 2.0.2.

**Comment by H4wKs** [comment]
Huge Thanks for your help with this matter  @shadowhand !

**Pull request merged**